content/en/contribute/medic/onboarding/all-the-things.md

```diff
@@ -0,0 +1,140 @@
+* [Offline-First]({{% ref "/core/overview/offline-first" %}}) - for real
+* [Architecture of the CHT]({{% ref "core/overview/architecture" %}})
+* [Community](https://communityhealthtoolkit.org/) of people and organizations. This is where we document all the things about the CHT Framework. Bookmark it as you primary source of reference. 
+
+### Teams at Medic
+Meet the [Medic team](https://medic.org/team/)!
+* [Product]({{% ref "contribute/medic/onboarding/product-team" %}})
+* Programs
+* Research
+* Internal Operations
+* External Affairs
+
+### Lifecycle of a [CHT Application]({{% ref "/running-programs/#cht-lifecycle" %}}) being built
+1. Programs team starts relationship with an organization.
+1. Service designers and app developers figure out how they want their system to work.
+1. App developers take latest version of the CHT and build the app for the organization.
+1. Android flavor deployed to get branded app onto CHW devices / deployment.
+1. Dashboards are set up in Klipfolio, Superset or Grafana.
+1. Go!
+1. App developers make ongoing enhancements.
+1. App upgraded as new versions of CHT are available.
+
+### CHT Academy
+* [Fantastic way](https://academy.communityhealthtoolkit.org/) to understand how certain features of the CHT work.
+
+## Now, all the things…
+### People/Team
+
+#### Distributed team 
+* Without some effort, it’s easy for things to feel lonely or isolated.
+* Default to asynchronous communication.
+* Respect teammate timezones (including your own!).
+
+#### Expensify
+* Reach out to Internal Operations team for guidance on how to submit expenses and get refunded. When submitting expenses, follow up to make sure things get through.  
+
+#### Funding
+* Restricted - Clicktime. Getting our ClickTime timesheets submitted on time is vital to source the projects adequately. If necessary, take extra steps to remind yourself of submitting these on a monthly basis. If you have any question about filling in the reports properly, reach out to the finance team.
+* Unrestricted
+
+#### Travel 
+* Team Meetups are a great way to build relations with your team! These are usually planned weeks ahead; if you feel comfortable joining, please do!
+* Focused Working Groups team members regurarly organize in-person meetups to meet with the people they serve. It's highly recommended to join those trips to get more connected to the team and the mission! 
+
+#### Meetings
+* There a few calls where you will be required to join. We know that depending on your timezone, you might need to adjust your calendar to be able to attend and we provide great flexibility to do so. 
+* Organization-wide calls are recorded.
+* No meetings on Fridays, as we consider Fridays as Deep Work days! 
+* Retrospective sessions
+* Daily Standups available in 2 timezones
```

Comment:
I hadn't read this in a while and doing so now made me think we're asking people to attend both.

```suggestion
* Daily Standups available in 2 timezones (pick the one that suits you best)
```